server/index.js

```diff
@@ -329,17 +320,11 @@ const loop = () => {
     SI.vault.add(snapshot);
     io.emit("playersUpdate", snapshot);
   }
-};
-setInterval(loop, 1000 / 30);
```

Comment:
Is there any reason to not update specific time bound values in separate loops? Main game loop is already pretty chaotic.